iob.bat

```diff
@@ -17,10 +17,18 @@ if [%1]==[fix] (
                     node node_modules/iobroker.js-controller/iobroker.js %1 %2 %3 %4 %5 %6 %7 %8
                 )
             ) else (
-                node node_modules/iobroker.js-controller/iobroker.js %1 %2 %3 %4 %5 %6 %7 %8
+				if [%1]==[restart] (
+					if [%2]==[] (
+						call serviceIoBroker.bat restart
+					) else (
+						node node_modules/iobroker.js-controller/iobroker.js %*
+					)
+				) else (
+					node node_modules/iobroker.js-controller/iobroker.js %1 %2 %3 %4 %5 %6 %7 %8
```

Comment:
basically approve, but can we also change the other calls with single parameters to use `%*`